src/components/Footer/Footer.tsx

```diff
@@ -1,15 +1,27 @@
 import React from 'react';
 
-import { Text } from '@chakra-ui/react';
+import { VStack } from '@chakra-ui/react';
 
 import { MaxWidthWrapper } from '@components/Misc';
 
 import { FooterContainer } from './FooterContainer';
+import { LinksSection } from './LinksSection';
+import { LogosSection } from './LogosSection';
+import { NewsletterSection } from './NewsletterSection';
 
 export const Footer = () => (
```

Comment:
I think chakra-ui is already contemplating the accessibility part but this is an interesting example of an accessible Footer,  you can take a look: https://a11y-style-guide.com/style-guide/section-navigation.html#kssref-navigation-navigation-footer 
There you can see that is using the correct semantic for the Footer and the lists inside it.